apps/site/components/Downloads/Release/ReleaseCodeBox.tsx

```diff
@@ -28,14 +29,32 @@ const ReleaseCodeBox: FC = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [release.versionWithPrefix, os, platform]);
 
+  const parseVersion = () => {
+    const version = release.versionWithPrefix.replace('v', '');
+    const versionParts = version.split('.');
+    return {
+      major: parseInt(versionParts[0]),
+      minor: parseInt(versionParts[1]),
+      patch: parseInt(versionParts[2]),
+    };
+  };
+
   const codeLanguage = os === 'WIN' ? 'PowerShell' : 'Bash';
 
   return (
     <div className="mb-2 mt-6 flex flex-col gap-2">
+      {parseVersion().major < 18 && (
+        <div
+          className="border-danger-400 bg-danger-100 relative rounded border px-4 py-3 text-red-700"
+          role="alert"
+        >
+          {t('layouts.download.codeBox.unsupportedVersionWarning')}&nbsp;
+          <a href={`/${locale}/about/previous-releases/`}>Previous Releases</a>
```

Comment:
You use `import Link from '@/components/Link';` to automatically use right local (so no need to use the `useLocal` hook)